### ISSUE TITLE
Error solving cache

### ISSUE THREAD

**Issue by wathhr** [peterkruse/DiscordMassAccountCreator#3]
When launching main.py I get this, https://wather.is-inside.me/RCSmzTsi.png even after setting the 2captcha api key on line 5. How could I fix this?

**Comment by peterkruse** [comment]
Hi.

That looks weird, havent got that error before, i will check it out and try to fix the error. 

Thanks for reporting it to me, again this is still in development and i am trying to fix all errors that may occur :D

**Comment by wathhr** [comment]
> Hi.
> 
> That looks weird, havent got that error before, i will check it out and try to fix the error.
> 
> Thanks for reporting it to me, again this is still in development and i am trying to fix all errors that may occur :D

Yeah, no worries! You can take your time, it's understandable considering how new this this is. Thanks for the reply!

**Comment by peterkruse** [comment]
I will try to found a solution as quick as possible, be welcome to make a pull request if you fix the problem 👍🏼

**Comment by wathhr** [comment]
Any progress so far?

**Comment by swendywashere** [comment]
when will the program improve?

**Comment by peterkruse** [comment]
I am planning to roll out a new update including recaptcha solving method maybe tomorrow, if everything goes fine

**Comment by wathhr** [comment]
> I am planning to roll out a new update including recaptcha solving method maybe tomorrow, if everything goes fine

Cool! Looking forward to it!

**Comment by peterkruse** [comment]
Hi.

Sorry for not updating the project, i have been working on alot other projects. I dont think when i exactly will push out a new update or maybe a working versions. 

I have a few things to test and add some features for a better program performance. Feel free to contribute to the project :D
